src/resdk/tables/base.py

```diff
@@ -536,7 +547,9 @@ async def _download_data(self, data_type: str) -> pd.DataFrame:
             source_urls = self._get_data_urls(uri_to_id.keys())
             urls_ids = [(url, uri_to_id[uri]) for uri, url in source_urls.items()]
 
-            async with aiohttp.ClientSession() as session:
+            async with aiohttp.ClientSession(
+                cookies=self.resolwe.auth.cookies
```

Comment:
This new session is not authenticated. If we do not set the right cookies the "Permission denied" is thrown (because now, we're trying to fetch something from https:aapp.genialis.com/ and we need authenticatioon for that.). 

Why is this added just now? Before, we could just follow the pre-signed urls, which work regardless of authentication.